src/components/screens/ScoreSetEditor.vue

```diff
@@ -897,16 +915,14 @@ export default {
     },
 
     fetchTargetAccessions: async function (event) {
-      if (this.targetAutocomplete == 'Assembly') {
-        const assembly = this.assembly?.trim() || null
-        if (assembly) {
-          this.targetGeneAccessionSuggestions = await this.fetchTargetAccessionsByAssembly(assembly)
+      if (this.targetAutocomplete == 'Assembly' ) {
```

Comment:
I would probably write a watcher for this.targetautocomplete to do the switching here.

Reply:
ok to skip